config/default.yml

```diff
@@ -1172,10 +1172,13 @@ Metrics/MethodLength:
 Metrics/BlockLength:
   CountComments: false  # count full line comments?
   Max: 25
-  Exclude:
-    - 'Rakefile'
-    - '**/*.rake'
-    - 'spec/**/*.rb'
+  ExcludedMethods:
```

Comment:
I wouldn't add library specific stuff to the default config. This should be in the RuboCop-specific config.